iceberg/iceberg-rest-server/src/test/java/org/apache/gravitino/iceberg/integration/test/IcebergRESTServiceIT.java

```diff
@@ -526,4 +519,42 @@ void testSnapshot() {
         convertToStringMap(sql("select * from iceberg_rest_table_test.snapshot_foo1"));
     Assertions.assertEquals(ImmutableMap.of("1", "a", "2", "b"), result);
   }
+
+  @Test
+  @EnabledIf("catalogTypeNotMemory")
+  void testRegisterTable() {
+    String registerDB = "iceberg_register_db";
+    String registerTableName = "register_foo1";
+    sql("CREATE DATABASE " + registerDB);
+    sql(
+        String.format(
+            "CREATE TABLE %s.%s (id bigint COMMENT 'unique id',data string) using iceberg",
+            registerDB, registerTableName));
+    sql(String.format("INSERT INTO %s.%s VALUES (1, 'a')", registerDB, registerTableName));
+
+    // get metadata location
+    List<String> metadataLocations =
+        convertToStringList(
+            sql(
+                String.format(
+                    "select file from %s.%s.metadata_log_entries", registerDB, registerTableName)),
```

Comment:
Can you please make all the SQL keywords uppercase.

Reply:
done